### PR TITLE
New Window Managers CheatSheet

### PR DESCRIPTION
**Pull request by laorynas**
## Description of new Instant Answer, or changes

Window Managers is list of Window Managers and their descriptions.

## People to notify
@laorynas

---

Instant Answer Page: https://duck.co/ia/view/window_managers

### CODE REVIEW

**Comment by daxtheduck** [comment]
@laorynas Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/window-managers.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
+{
+    "id": "window_managers_cheat_sheet",
+    "name": "Window Managers",
+    "description": "Window Managers is list of Window Managers and their descriptions",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/window-managers.json]: Please change this to something descriptive:  `A window manager is system software that controls the placement and appearance of windows within a windowing system in a graphical user interface.`

File: share/goodie/cheat_sheets/json/window-managers.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            },
+            {
+                "key": "Fluxbox",
+                "val": "Fluxbox is a windowmanager for X that was based on the Blackbox 0.61.1 code. It is very light on resources and easy to handle but yet full of features to make an easy, and extremely fast, desktop experience. It is built using C++ and licensed under the MIT-License"
+            },
+            {
+                "key": "FLWM",
+                "val": "The Fast Light Window Manager is a stacking window manager written in C++ and available for redistribution under the terms of the GNU General Public Licence. FLWM is the default window manager for Tiny Core Linux"
+            },
+            {
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/window-managers.json]: Lets split it here...
Add a new section named `Active 2`

File: share/goodie/cheat_sheets/json/window-managers.json [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+    "template_type": "reference",
+
+    "section_order": [
+        "Active",
+        "Discontinued"
+    ],
+
+    "sections": {
+        "Active": [
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/window-managers.json]: Change this to `Active 1`

**Comment by laorynas** [comment]
Ok I will do the changes when I have some free time.

File: share/goodie/cheat_sheets/json/window-managers.json (hunk 2 above)
Comment (laorynas) [inline share/goodie/cheat_sheets/json/window-managers.json]: Done the changes

File: share/goodie/cheat_sheets/json/window-managers.json [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+    "sections": {
+        "Active 1": [
+            {
+                "key": "AfterStep",
+                "val": "AfterStep is a stacking window manager for the X Window System. The goal of AfterStep's development is to provide for flexibility of desktop configuration, improved aesthetics and efficient use of system resources, and was used in such distributions as MachTen"
+            },
+            {
+                "key": "Awesome",
+                "val": "awesome is a highly configurable, next generation framework window manager for X. It is very fast, extensible and licensed under the GNU GPLv2 license"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/window-managers.json]: Start val field with a capital letter

File: share/goodie/cheat_sheets/json/window-managers.json [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+                "key": "SLWM",
+                "val": "SLWM is a new concept in window management. Its goal is to be a hybrid of a tiled and regular window manager, with the addition of an infinite desktop. All windows are also headless"
+            },
+            {
+                "key": "WMFS",
+                "val": "Window Manager From Scratch, A tiling window manager highly configurable"
+            },
+            {
+                "key": "wmii",
+                "val": "window manager improved is a small, scriptable window manager for X11, with a 9P filesystem interface and an acme-like layout"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/window-managers.json]: Start val field with a capital letter

Comment (laorynas) [inline share/goodie/cheat_sheets/json/window-managers.json]: Sorry for delay.
Only value with capital letter or key to?

**Comment by laorynas** [comment]
Waiting for response

**Comment by gautamkrishnar** [comment]
@jophab Please review this. I will merge this once it is reviewed.....

File: share/goodie/cheat_sheets/json/window-managers.json [hunk 6]
```diff
@@ -0,0 +1,120 @@
+{
+    "id": "window_managers_cheat_sheet",
+    "name": "Window Managers",
+    "description": "A window manager is system software that controls the placement and appearance of windows within a windowing system in a graphical user interface",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/window-managers.json]: The following starting would be better
"Window manager is a  system......"

**Comment by jophab** [comment]
@laorynas @gautamkrishnar This LGTM :+1:

**Comment by daxtheduck** [comment]
@laorynas this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by gautamkrishnar** [comment]
👍 Merged... Thanks for your contribution...

**Comment by moollaza** [comment]
@laorynas @gautamkrishnar please be more careful. The ID for this (and the IA Page) are both incorrect.

The ID MUST contain `_cheat_sheet`. The IA Page should have "Cheat Sheet" in the title as well.

I've corrected the ID to https://duck.co/ia/view/window_managers_cheat_sheet

**Comment by gautamkrishnar** [comment]
@moollaza Sorry for that. I forgot It. @jophab  Please take a note on this one...

**Pull request merged**